testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestComments.java

```diff
@@ -154,7 +156,12 @@ public void testCommentColumn()
                         "CREATE TABLE hive.default.%s (\n" +
                         "   c1 bigint COMMENT 'test comment',\n" +
                         "   c2 bigint COMMENT '',\n" +
-                        "   c3 bigint\n" +
+                        "   c3 bigint,\n" +
+                        "   c4 bigint COMMENT 'test partition comment'\n" +
+                        ")\n" +
+                        "WITH (\n" +
+                        "   format = 'RCBINARY',\n" +
+                        "   partitioned_by = ARRAY['c4']\n" +
```

Comment:
Is the following `\\E(?s:.*)` pattern still required? Same for other places.

Reply:
This is an existing code. Please let me know if you want me to improve it.